featuretools/__init__.py

```diff
@@ -16,3 +16,20 @@
 from .feature_base import AggregationFeature, DirectFeature, Feature, FeatureBase, IdentityFeature, TransformFeature
 
 __version__ = '0.6.1'
+
+import pkg_resources
+import sys
+for entry_point in pkg_resources.iter_entry_points('featuretools_initialize'):
```

Comment:
add a comment to this block? something like

"Call functions registered by other libraries when featuretools is imported"

Reply:
Added